### PR TITLE
fix: consuming node status from diagnosis-config-get-iter zapi for 7mode

### PR DESCRIPTION
**Pull request by Hardikl**
using diagnosis-config-get-iter zapi to get node status with monitor type as node-connect

image

### CODE REVIEW

File: conf/zapi/7mode/8.6.0/status_7.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  diagnosis-config-info:
+    - ^health                       => health
+    - ^^monitor                     => monitor
+    - mon-version                   => monitor_version
+
+collect_only_labels: true
+
+plugins:
+  - LabelAgent:
+    value_mapping: 7mode_status health ok `1`
```
Comment (cgrinds) [inline conf/zapi/7mode/8.6.0/status_7.yaml:17]: new code should not use the deprecated `value_mapping`

File: conf/zapi/7mode/8.6.0/status_7.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    # metric label zapi_value rest_value `default_value`
+    value_to_num: 7mode_new_status health ok todo `0`
```
Comment (cgrinds) [inline conf/zapi/7mode/8.6.0/status_7.yaml:19]: with `value_mapping` gone does it make more sense to name this `7mode_status`

File: grafana/dashboards/7mode/harvest_dashboard_cluster7.json [hunk 3]
```diff
@@ -236,7 +236,7 @@
         },
         {
           "exemplar": false,
-          "expr": "cluster_new_status{datacenter=\"$Datacenter\",cluster=\"$Cluster\"}",
+          "expr": "node_7mode_new_status{datacenter=\"$Datacenter\",cluster=\"$Cluster\",monitor=\"node-connect\"}",
```
Comment (cgrinds) [inline grafana/dashboards/7mode/harvest_dashboard_cluster7.json:239]: `node_7mode_status`

File: grafana/dashboards/7mode/harvest_dashboard_cluster7.json [hunk 4]
```diff
@@ -418,7 +418,7 @@
       "targets": [
         {
           "exemplar": false,
-          "expr": "cluster_new_status{datacenter=~\"$Datacenter\",cluster=\"$Cluster\"}",
+          "expr": "node_7mode_new_status{datacenter=\"$Datacenter\",cluster=\"$Cluster\", monitor=\"node-connect\"}",
```
Comment (cgrinds) [inline grafana/dashboards/7mode/harvest_dashboard_cluster7.json:421]: `node_7mode_status`

File: conf/zapi/7mode/8.6.0/status_7.yaml (hunk 1 above)
Comment (Hardikl) [reply conf/zapi/7mode/8.6.0/status_7.yaml:17]: agreed.

File: conf/zapi/7mode/8.6.0/status_7.yaml (hunk 2 above)
Comment (Hardikl) [reply conf/zapi/7mode/8.6.0/status_7.yaml:19]: yes.

File: grafana/dashboards/7mode/harvest_dashboard_cluster7.json (hunk 3 above)
Comment (Hardikl) [reply grafana/dashboards/7mode/harvest_dashboard_cluster7.json:239]: done

File: grafana/dashboards/7mode/harvest_dashboard_cluster7.json (hunk 4 above)
Comment (Hardikl) [reply grafana/dashboards/7mode/harvest_dashboard_cluster7.json:421]: done

**Review by cgrinds — APPROVED** [review]
thanks @Hardikl

**Pull request merged**